app/api/orders.py

```diff
@@ -324,6 +328,21 @@ def before_update_object(self, order, data, view_kwargs):
             if order.order_notes and data['order_notes'] not in order.order_notes.split(","):
                 data['order_notes'] = '{},{}'.format(order.order_notes, data['order_notes'])
 
+        status = data.get('status', None)
+        if status and status not in ['cancelled', 'completed', 'expired', 'initializing', 'pending', 'placed']:
+            raise ConflictException({'pointer': '/data/attributes/status'},
+                                    "status '{}' is invalid".format(status))
```

Comment:
please remove inner quotes for `'{}'`

Reply:
Actually I wanted to have error like this - `status 'abcd' is invalid?`
However I think this isn't necessary. Updating it right away :)